dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/event/DefaultEventService.java

```diff
@@ -120,7 +122,8 @@ private FileResource getFileResourceMetadata(UID eventUid, UID dataElementUid)
               .orgUnitMode(OrganisationUnitSelectionMode.ACCESSIBLE)
               .events(Set.of(eventUid))
               .eventParams(EventParams.FALSE)
-              .dataElementFilters(Map.of(dataElementUid, List.of()))
+              .dataElementFilters(
```

Comment:
Does

`.dataElementFilters(Map.of(dataElementUid, List.of()))` still work but 

`Map.of(dataElementUid, List.of(new QueryFilter(QueryOperator.NNULL))))` is more clear in describing what we want here?

Reply:
`.dataElementFilters(Map.of(dataElementUid, List.of()))` is something I introduced in part 1 of this PR as a temporary step.

In this PR, on [the web layer](https://github.com/dhis2/dhis2-core/pull/19778/files#diff-821fe85408531050058036e5a373fee939b3ca9f8ed35c8f09a7681f59a4c50cR406) I'm adding the `!null` operator in case a filter has a UID only, so we can't have an empty list as a value of this map on the service layer anymore.